src/Parser/Hunter/Marksmanship/Modules/Features/AlwaysBeCasting.js

```diff
@@ -1,45 +1,79 @@
 import React from 'react';
 
 import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+import Combatants from 'Parser/Core/Modules/Combatants';
 
 import SPELLS from 'common/SPELLS';
 import { formatPercentage } from 'common/format';
 import { STATISTIC_ORDER } from 'Main/StatisticBox';
 import SpellLink from 'common/SpellLink';
 
 class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static dependencies = {
+    combatants: Combatants,
```

Comment:
You should put this below the other line or it won't properly use an extended version.